src/main/java/foxtrot/uniform/charlie/kilo/model/dto/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,40 @@
+package foxtrot.uniform.charlie.kilo.model.dto;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private Long userId;
```

Comment:
see checklist saying about ShoppingCartResponseDto

Reply:
I saw it but didn't understood. I refactored it but still not sure.